test/tests/kernels/hfem/neumann.i

```diff
@@ -0,0 +1,99 @@
+[Mesh]
+  [square]
+    type = GeneratedMeshGenerator
+    nx = 3
+    ny = 3
+    dim = 2
+  []
+  build_all_side_lowerd_mesh = true
+[]
+
+[Variables]
+  [u]
+    order = THIRD
+    family = MONOMIAL
+    block = 0
+  []
+  [lambda]
+    order = CONSTANT
+    family = MONOMIAL
+    block = INTERNAL_SIDE_LOWERD_SUBDOMAIN
+  []
+[]
+
+[AuxVariables]
+  [v]
```

Comment:
We do not need this aux var if we replace `CoupledForce` with `BodyForce` kernel.